editions/tw5.com/tiddlers/widgets/WikifyWidget.tid

```diff
@@ -28,3 +28,15 @@ The available output types are:
 |html |Return the rendered HTML of the output |
 |parsetree |Return a formatted JSON representation of the parse tree |
 |widgettree |Return a formatted JSON representation of the widget tree |
+
+The widget is mostly used to generate [[WikiText]] and [[Filters]]: for example, a filter that is built from list of tags could look like this:
+
+<$macrocall $name="wikitext-example-without-html" src="""<$vars listOfTags="Concepts Reference">
+<$wikify name="tagsFilter" text="[<$list filter='[enlist<listOfTags>]' variable='currentTag'>tag[<<currentTag>>]</$list>first[5]]">
```

Comment:
This is not how to do things. The entries in the list of tags will end up being wikified! Try it with a tag called `a//b//c`. What is the goal of this sample?

Reply:
From a list of given tags build a filter that lists the tiddlers that are tagged with each tag.

Reply:
I found where I saw this technique: 
https://groups.google.com/forum/#!msg/tiddlywiki/nyGIYgun2s8/h7E6q0lNAgAJ

Reply:
Thanks @bimlas